ship.config.js

```diff
@@ -20,10 +19,9 @@ module.exports = {
   beforeCommitChanges: ({ exec }) => {
     exec('yarn doctoc');
   },
-  pullRequestTeamReviewer: ['instantsearch-for-websites'],
+  pullRequestTeamReviewers: ['instantsearch-for-websites'],
   buildCommand: ({ version }) =>
     `NODE_ENV=production VERSION=${version} yarn build`,
-  testCommandBeforeRelease: () => 'echo "No need to test again."',
```

Comment:
The default behavior of the latest shipjs skips this test step, so we don't need this any more.